### DISCUSSION TITLE
Error when opening the hub

### DISCUSSION THREAD

**Discussion by roblox-afk** [Redon-Tech/Roblox-Purchasing-Hub#18] [Q&A] [answered]
when i play the hub it is saying that and error has occurred: Remote Server Offline

![image](https://user-images.githubusercontent.com/66645833/154858211-4c7820e5-4aff-40de-ba27-71ca0c0d95e6.png)

**Comment by roblox-afk** [comment]
Its also saying this in the console

![image](https://user-images.githubusercontent.com/66645833/154858422-e1c37cfe-fce8-4e68-b30e-af860846dd2f.png)

**Comment by parker02311** [comment, marked as answer]
Miss-configured mongodb server
